spec/duration.html

```diff
@@ -169,31 +157,11 @@ <h1>Temporal.Duration.from ( _arg_ )</h1>
       </p>
       <emu-alg>
         1. If Type(_arg_) is String, then
-          1. Return ? DurationFromString(_arg_).
+          1. Let _isoString_ be ? ToString(_arg_).
+          1. Return ? DurationFromString(_isoString_).
         1. If Type(_arg_) is Object, then
-          1. If _arg_ has an [[InitializedTemporalDuration]] internal slot, then
-            1. Return _arg_.
-          1. Let _result_ be the Record {
-            [[Years]]: *undefined*,
-            [[Months]]: *undefined*,
-            [[Days]]: *undefined*,
-            [[Hours]]: *undefined*,
-            [[Minutes]]: *undefined*,
-            [[Seconds]]: *undefined*,
-            [[Milliseconds]]: *undefined*,
-            [[Microseconds]]: *undefined*,
-            [[Nanoseconds]]: *undefined*,
-          }.
-          1. For each row of <emu-xref href="#table-temporal-durationlike-properties"></emu-xref>, except the header row, in table order, do
-            1. Let _prop_ be the Property value of the current row.
-            1. Let _val_ be ? Get(_arg_, _prop_).
-            1. If _val_ is *undefined*, then
-              1. Let _val_ be *0*.
-            1. Else
-              1. Let _val_ be ? ToInteger(_val_).
-            1. Set _result_'s internal slot whose name is the Internal Slot value of the current row to _val_.
-          1. Return ? CreateDuration(_result_.[[Years]], _result_.[[Months]], _result_.[[Days]], _result_.[[Hours]], _result_.[[Minutes]], _result_.[[Seconds]], _result_.[[Milliseconds]], _result_.[[Microseconds]], _result_.[[Nanoseconds]]).
-        1. Throw a *RangeError* exception.
+          1. Return ? ToDuration(_arg_).
+        1. Throw a *TypeError* exception.
```

Comment:
Remove everything except the `ToDuration` call.

Reply:
This should ideally be dealt with in #269.